index.html

```diff
@@ -310,32 +307,73 @@ <h2>contact me</h2>
                         to help with it.
                     </p>
                 </div>
-                <div class="contact-me-form">
-                    <form class="form" action="https://formspree.io/f/mvoyejav" method="post">
-                        <div class="username-input">
-                            <label hidden id="mylabel" for="username">Username</label>
-                            <input type="text" name="username" class="input" placeholder="enter username" maxlength="30"
-                                required />
+                <div 
+                    class="contact-me-form">
+                    <form class="form" 
+                    action="https://formspree.io/f/mvoyejav" 
+                    id="form"
+                    method="post"
+                    onsubmit="event.preventDefault(); validateForm()">
+
+                        <div 
+                        class="username-input">
+                         <label hidden 
+                            id="mylabel" 
+                            for="name">Name</label>
+                            <input 
+                            type="text" 
+                            name="name" 
+                            class="input" 
+                            id="name"
+                            placeholder="name" 
+                            maxlength="30"
+                            required />
+                            <small class="error"></small>
                         </div>
                         <br />
                         <div class="email-input">
-                            <label hidden id="mylabel" for="email">Email</label>
-                            <input type="email" name="email" class="input" placeholder="enter email" size="64"
-                                maxlength="64" required />
+                            <label 
+                            hidden 
+                            id="mylabel" 
+                            for="email">
+                            Email
+                        </label>
+                            <input 
```

Comment:
- [ ] It would be better if our email checks pass when all our letters are lowercase letters.However, we still get error as seen in the screenshot below
![Screenshot from 2023-01-05 21-34-33](https://user-images.githubusercontent.com/46388113/210883667-750aac83-a113-43d6-96c4-d60120b5b733.png)